release.sh

```diff
@@ -48,9 +48,9 @@ package() {
 	
 	if [[ $1 == "darwin" ]]; then
 		mv "$ui/Electron.app" "$ui/Sia.app"
-		rm -r "$ui/Sia.app/Contents/Resources/default_app"
-		cp -R ../app/ "$ui/Sia.app/Contents/Resources/app/"
-		mkdir -p "$ui/Sia.app/Contents/Resources/app/Sia"
+		rm -rf "$ui/Sia.app/Contents/Resources/default_app"
+		rsync -av --progress ../ "$ui/Sia.app/Contents/Resources/app/" --exclude "release/" --exclude ".git/"
+		mkdir -p "$ui/Sia.app/Contents/Resources/app/app/Sia"
```

Comment:
app/app is definitely not okay. app/ui is also not okay, why do we need this double-layering when before we didn't?

This is the reason that I removed electron-packager before, I couldn't figure out how to not have it do a double-app folder in the release. Changing the name is just masking the problem, we want as simple of a directory structure as possible.